mesonbuild/build.py

```diff
@@ -19,17 +19,18 @@
 import hashlib
 import pickle
 from functools import lru_cache
+from typing import Dict, List
```

Comment:
__[Flake8]__ [F811] redefinition of unused 'List' from line 15 ([link](https://sider.review/gh/repos/19784232/pulls/4010#issue-4029504))

<sub>posted by Sider</sub>